inst/python/dp/custom_dp/covariance_custom.py

```diff
@@ -108,9 +107,7 @@ def custom_bounded_covariance(
     xy_normalized = (x - noisy_average_x) * (y - noisy_average_y)
 
     # Compute the noisy average of the normalized product, ie the covariance
-    noisy_average_xy = noisy_average(
-        xy_normalized, eps_per_query, xy_min, xy_max, n, n_noisy
-    )
+    noisy_average_xy = noisy_average(xy_normalized, eps_per_query, xy_min, xy_max, n, n_noisy)
```

Comment:
As discussed, linter/ syntax validator would be cool to avoid this!

Reply:
Yes! It's part of this work :) 

https://github.com/arkhn/dsPrivacy/pull/14